storage/tests/unit/test_blob.py

```diff
@@ -1612,23 +1619,25 @@ def _upload_from_file_helper(self, side_effect=None, **kwargs):
         num_retries = kwargs.get('num_retries')
         blob._do_upload.assert_called_once_with(
             client, stream, content_type,
-            len(data), num_retries, predefined_acl)
+            len(data), num_retries, predefined_acl,
+            extra_headers)
         return stream
 
     def test_upload_from_file_success(self):
-        stream = self._upload_from_file_helper(predefined_acl='private')
+        stream = self._upload_from_file_helper(
+            predefined_acl='private', extra_headers={})
```

Comment:
For clarity, please pass an actual header here, e.g., `extra_headers={'X-Goog-Testing': 'true'}`.